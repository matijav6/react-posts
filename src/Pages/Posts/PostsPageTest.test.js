import renderer from 'react-test-renderer';
import {PostsPage} from './';

test('Posts page renders correctly', () => {
    let wrapper = renderer.create(<PostsPage hello={""} sendRequest={()=>{}}/>);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
});
