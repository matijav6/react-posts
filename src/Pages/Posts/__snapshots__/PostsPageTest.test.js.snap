// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts page renders correctly 1`] = `
Array [
  <nav
    className="navbar navbar-dark bg-primary d-flex justify-content-center min-height-50"
  >
    <form
      className="form-inline w-75"
    >
      <input
        className="form-control mr-sm-2 w-100"
        onChange={[Function]}
        placeholder="Search"
        type="text"
      />
    </form>
  </nav>,
  <div
    className="container-fluid p-3"
  >
    <div
      className="row justify-content-md-center"
    />
  </div>,
]
`;
